types/braintree-web/package.json

```diff
@@ -13,10 +13,6 @@
         "@types/braintree-web": "workspace:."
     },
     "owners": [
-        {
-            "name": "Jason Buckner",
```

Comment:
why remove Jason Buckner?

Reply:
The commit message explains it: "Unfortunately jbuckner's inactivity of 4 years is slowing us other owners down."